resolvers/utils/helpers.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-param-reassign */
+const {
+  ARTS_KEYWORDS,
+  BUSINESS_KEYWORDS,
+  HEALTH_KEYWORDS,
+  INTERDISCIPLINARY_KEYWORDS,
+  PUBLIC_SOCIAL_SERVICES_KEYWORDS,
+  COMP_TECH_KEYWORDS,
+  STEM_KEYWORDS,
+  SOCIAL_SCIENCES_KEYWORDS,
+  TRADES_KEYWORDS,
+} = require('./keywords');
+
+/**
+ * This populates the metrics object where the key matches what the hacker put
+ * as their value.
+ *
+ * @param {object} metrics
+ * @param {object} hacker
+ */
+function populateIndependents(metricSet, key) {
```

Comment:
I was able to make this extremely simple as opposed to using a switch statement. It will basically just populate a set object with keys that match the values that the hackers picked from.

This will also be easy to debug with because if we see a new key inside the metrics object, then we know something is up with the csv. I actually found a few typos inside of one.